src/wovnio/html/HtmlConverter.php

```diff
@@ -0,0 +1,155 @@
+<?php
+namespace Wovnio\Html;
+
+/**
+ * Convert html via Simple HTML DOM Parser
+ *
+ * @see http://simplehtmldom.sourceforge.net/manual.htm
+ */
+class HtmlConverter {
+  private $html;
+  private $token;
+
+  /**
+   * HtmlConverter constructor.
+   * @param String $html
+   * @param String $token project_token
+   */
+  public function __construct($html, $token) {
+    $this->html = $html;
+    $this->token = $token;
+  }
+
+  /**
+   * Convert to appropriate HTML to send to Translation API
+   * e.g.) remove wovn-ignore content
+   *
+   * @return array converted html and HtmlReplaceMarker
+   */
+  public function convertToAppropriateForApiBody() {
+    // encoding detection uses 30% of execution time for this method, should we make encoding as setting?
+    $encoding = mb_detect_encoding($this->html, array('UTF-8', 'EUC-JP', 'SJIS', 'eucJP-win', 'SJIS-win', 'JIS', 'ISO-2022-JP', 'ASCII'));
```

Comment:
I agree. If user always use same encoding, they should be able to specify settings for faster execution. I user does not specify encoding, we should keep detection.

If we do that, we should also document special setup for optimization purpose (not related bit need to do same with curl library).

Reply:
done. `encoding` option is created

Reply:
thanks